SonarQube.MSBuild.Tasks/Targets/SonarQube.Integration.targets

```diff
@@ -364,6 +364,10 @@
       <CodeAnalysisForceOutput>true</CodeAnalysisForceOutput>
       <CodeAnalysisGenerateSuccessFile>true</CodeAnalysisGenerateSuccessFile>
       <CodeAnalysisIgnoreGeneratedCode>true</CodeAnalysisIgnoreGeneratedCode>
+
+      <!-- Make sure no warnings are treated as errors -->
+      <TreatWarningsAsErrors>false</TreatWarningsAsErrors>
+      <WarningsAsErrors></WarningsAsErrors>
```

Comment:
We have separate targets that set the properties for FxCop and Roslyn analysis. I've set the appropriate properties in both sections.